### DISCUSSION TITLE
Delayed Access to Student Developer Pack: Seeking Assistance and Resolution

### DISCUSSION THREAD

**Discussion by dikshant-pandey1** [community/community#60013] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone,

I hope you're doing well. I'm currently experiencing an issue with accessing the Student Developer Pack, despite renewing my student verification and receiving approval. I renewed my verification yesterday, and it was successfully approved. However, I have not yet gained access to the Student Developer Pack, even though more than 24 hours have passed since the approval.

Whenever I try to access the pack, I receive a message stating, "Your Student Developer Pack will be ready soon! We will notify you via email when your Experiences and offers become available to you." However, I haven't received any notification or update regarding the activation of my benefits.

I want to highlight that this is not my first time renewing my student verification or accessing the Student Developer Pack. In the past, I have been able to access the pack immediately after the approval. This is the first time I have encountered such a delay, and it's causing inconvenience as I rely on the resources and opportunities provided by the pack for my learning and development on GitHub.

I have checked my email and spam folders, but I haven't received any notifications regarding the availability of the Student Developer Pack. I would appreciate any assistance or guidance from the GitHub community and support team to resolve this issue and gain access to the pack as soon as possible.

Thank you for your attention, and I look forward to your response and assistance.

Best regards,
Dikshant Pandey

**Comment by queenofcorgis** [comment]
I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
